build/theater.js

```diff
@@ -223,10 +223,9 @@
 
         if (typeof scene === "string") {
           var params   = scene.split(":"),
-              hasActor = params.length > 1,
+              hasActor = (params.length > 1 & params[0][params[0].length-1] !== '#'),
```

Comment:
The backslash is commonly used to escape special characters, `\:` is more appropriate than `#:`